### ISSUE TITLE
Invalid dependency when using corworkers 0.6.1

### ISSUE THREAD

**Issue by melalj** [tjmehta/coworkers-test#2]
I get this error message when I try to install coworkers-test:

```
npm ERR! peerinvalid Peer coworkers-test@0.1.0 wants coworkers@^0.3.0
```

**Comment by tjmehta** [comment]
Hmm, do you know how to fix this? The semver pattern appears correct. Any ideas?

**Comment by melalj** [comment]
It looks like the versions are limited to 0.3.x and not to 0.x.x
following: https://docs.npmjs.com/misc/semver#caret-ranges-123-025-004

You can edit the package.json to:

```
"peerDependencies": {
    "coworkers": "^0.6"
  },
```

and 

```
"devDependencies": {
    "coworkers": "^0.6"
  },
```

**Comment by tjmehta** [comment]
k think I fixed this: `coworkers-test@v0.1.1`
